src/shared/components/mutationMapper/MutationMapperUtils.ts

```diff
@@ -32,3 +34,39 @@ export function createVariantAnnotationsByMutationFetcher(
         }
     };
 }
+
+export function buildNamespaceColumnConfig(
+    mutations: Mutation[]
+): NamespaceColumnConfig {
+    if (!mutations) {
+        return {};
+    }
+    const namespaceConfig: NamespaceColumnConfig = {};
+    const nameSpaces = _.flatMap(mutations, m => _.keys(m.namespaceColumns));
```

Comment:
it seems strange here that something of type Mutation[] would have something called "namespaceColumns" on it

Reply:
I do not see why that would be strange. Maybe `namespaceValues` would be better. But would we go through the motions in the frontend and backend to update this?

Reply:
Again not necessary to address in this PR but what I object to is A) having something called "column" on Mutation type.  And "namespace".    Shouldn't this be more generic?   Like, customValues or something.  The fact that they belong to namespaces seems like a property of the individual values themselves.   Again, I know this is very late to the game and I am not suggesting we make a change, but worth bringing up.  Adding @inodb here.  Is this THE one and only way we allow people to extend the Mutation data?